README.md

```diff
@@ -52,6 +52,19 @@ You can now push to the specified environment by running
 skaffold run
 ```
 
+## Connecting psql client using the Cloud SQL Proxy:
+Go to cloud console to get the instance name you want to connect, then start the proxy:
+```
+./cloud_sql_proxy -instances=<INSTANCE_CONNECTION_NAME>=tcp:5432
```

Comment:
I don't see a ./cloud_sql_proxy file in the PR.

Reply:
Added